src/components/PopoverMenu.tsx

```diff
@@ -186,7 +186,7 @@ function PopoverMenu({
         if (menuItems.length === 0) {
             return;
         }
-        setEnteredSubMenuIndexes([]);
+        setEnteredSubMenuIndexes(CONST.EMPTY_ARRAY);
         setCurrentMenuItems(menuItems);
     }, [menuItems]);
```

Comment:
This fixes the reassure extra render count. This effect is run on mount and on `menuItems` prop update. `enteredSubMenuIndexes` initial value is `[]` and we update it to `[]` which has different references. Using `CONST.EMPTY_ARRAY` so it has stable references.